db.tf

```diff
@@ -17,7 +17,7 @@ data "vault_generic_secret" "apiarydb_master_user" {
 }
 
 resource "aws_security_group" "db_sg" {
-  name   = "sg-apiary-db"
```

Comment:
Hmmm that's interesting. I guess I see now that it's meant to be the 'Name' tag, not the 'Group Name', they are two different things. Confusing because on an EC2 instance, the 'name' and the Name tag always change together.

Since we use a variable for the tags, this doesn't seem to apply anyway. I will approve this without change.